lib/Server.js

```diff
@@ -389,6 +389,23 @@ function Server(compiler, options) {
 			const pems = selfsigned.generate(attrs, {
 				algorithm: "sha256",
 				days: 30,
+				extensions: [{
+					name: "subjectAltName",
+					altNames: [
+						{
+							type: 2, // DNS
+							value: "localhost"
+						},
+						{
+							type: 6, // URI
+							value: "localhost"
+						},
+						{
+							type: 7, // IP
+							ip: "127.0.0.1"
+						}
```

Comment:
Another popular domain to use (especially when developing an app that makes use of subdomains) is http://lvh.me. Maybe we can add support for a wildcard domain *.lvh.me here?